sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -307,7 +307,7 @@ def repository_name(self) -> str:
 class GetManifestResult(object):
     """The get manifest result.
 
-    :ivar dict manifest: The manifest JSON.
+    :ivar MutableMapping[str, Any] manifest: The manifest JSON.
```

Comment:
Not sure the type hint will render correctly in docs like this. You might want to move it to the :vartype: